### DISCUSSION TITLE
about/

### DISCUSSION THREAD

**Discussion by giscus** [152334H/152334H.github.io#5] [Announcements]
# about/

https://152334h.github.io/about/

**Comment by ben-narrativ-ai** [comment]
Just wrote you a message on LinkedIn! Would love to speak with you about working on TorToise in a professional setting. Please feel free to respond to me there or via emailing me directly: ben@narrativ.ai

Thanks!

**Comment by flipkast** [comment]
Hi Sherman, Oliver here. I requested a LinkedIn connection with you. As the owner of a rapidly growing TTS-TTV platform that supports Tortoise for voice generation, I would be happy to discuss a possible collaboration with you. Thanks!
